### PR TITLE
remove duplicate tests from openshift-tests-kubernetes

### PR DESCRIPTION
**Pull request by sallyom**
* removed openshift-tests-kubernetes suite from extended-platform-tests
* pulled in recent changes to origin
* added origin/examples from master branch (some tests rely on examples)
* update Dockerfile to add extended-platform-tests to tests image (depends on https://github.com/openshift/release/pull/7215 ) to create an image:
  `registry.svc.ci.openshift.org/ocp/4.4:extended-platform-tests` that includes the `openshift-tests-kubernetes` + `extended-platform-tests` binaries - and for transition time also include the `openshift-tests` binary (this image will be renamed to become the release:tests image and the `openshift-tests` binary will be removed)

need to merge in this order:
1) ~https://github.com/openshift/origin/pull/24450~ https://github.com/openshift/origin/pull/24560
2) https://github.com/openshift/release/pull/7215  release PR to build tests-kubernetes image
2) this PR - will result in extended-platform-tests image w/ 3 binaries listed above
3) https://github.com/openshift/release/pull/7190 adds a new template to test the new binaries from extended-platform-tests image built here.
4) https://github.com/openshift/release/pull/6789 test job to openshift/release in any component to run new template, to confirm all suites run. 
5) rename extended-platform-tests image to become the release:tests (remove the current tests image build from ci-operator/config/openshfit/origin/*.yaml and rename image built in ci-operator/config/openshift/openshift-tests/*yaml
6) edit all ci-operator/openshfit/installer/* templates to run new binaries
7) remove `openshift-tests` binary from the image built in this repo. 
    

_this is how i understand what needs to happen_

/cc @deads2k

### CODE REVIEW

File: Dockerfile.rhel [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+WORKDIR /go/src/github.com/openshift/openshift-tests
 COPY . .
 RUN make build WHAT=cmd/extended-platform-tests && \
     mkdir -p /tmp/build && \
-    cp /go/src/github.com/openshift/extended-platform-tests/extended-platform-tests /tmp/build/extended-platform-tests
+    cp /go/src/github.com/openshift/openshift-tests/extended-platform-tests /tmp/build/extended-platform-tests
 
-FROM registry.svc.ci.openshift.org/ocp/4.3:cli
+# FROM registry.svc.ci.openshift.org/ocp/4.3.0:kubernetes-tests
+FROM registry.svc.ci.openshift.org/somalley1/stable:kubernetes-tests
```
Comment (sallyom) [inline Dockerfile.rhel]: I've been using ci ns `somalley1` to test with: registry.svc.ci.openshift.org/somalley1/4.3.0:kubernetes-tests

File: examples/service-catalog/OWNERS [hunk 2]
```diff
@@ -0,0 +1,9 @@
+reviewers:
+  - bparees
+  - pmorie
+  - jboyd01
```
Comment (jianzhangbjz) [inline examples/service-catalog/OWNERS:4]: @sallyom I guess the `jboyd01/jpeeler` left.

Comment (sallyom) [reply examples/service-catalog/OWNERS:4]: i copied this OWNERS from origin/examples. i'll check on it

**Comment by sallyom** [comment]
openshift-tests-kubernetes is going in a different direction, closing

**Pull request closed without merging**